userspace/chisel/chisel_api.cpp

```diff
@@ -194,6 +194,9 @@ uint32_t lua_cbacks::rawval_to_lua_stack(lua_State *ls, uint8_t* rawval, ppm_par
 				}
 			}
 			break;
+		case PT_CHARBUFARRAY:
```

Comment:
Totally fine with just skipping these for now, but maybe a comment saying that a complete solution would involve creating a lua table, populating it with the values, and pushing that onto the stack?